Add philanthropic fundraising (#672)
publiccodenet/about: activities/philanthropic-fundraising/donating-netherlands.md

---
type: Guide
---

# How to donate to us

> If you want to support us you can transfer your donation to our bank account: NL78 RABO 0342 6453 31 in the name of "Foundation for Public Code Vereniging" in Amsterdam stating "donation". More information on periodic donations work below.

Reviewer: ```suggestion
If you want to support us you can transfer your donation to our bank account: NL78 RABO 0342 6453 31 in the name of "Foundation for Public Code Vereniging" in Amsterdam stating "donation". 
```

> ## Tax deductability

Reviewer: ```suggestion
## Tax deductibility
```

> You may deduct a donation to an ANBI (public benefit organization) in your income tax return. However, the conditions are different for a periodic gift and an ordinairy gift.

Reviewer: ```suggestion
You may deduct a donation to an ANBI (public benefit organization) in your Dutch income tax return. However, the conditions are different for a repeated (periodic) gift and one-off (ordinary) gift.
```

### Periodic donation

Your donation is a periodic donation if it meets one of the following 3 conditions:

* You give the same amount annually to the same institution or association. You may decide for yourself whether you pay your annual donation in one go, or in instances.
* You pay the amount 5 years in a row. Longer is also allowed.
* You have determined when your annual donation stops. For example, after 5 years, or when you pass away.

> ### Ordinairy donation

Reviewer: ```suggestion
### Ordinary donation
```

> An ordinairy donation is a donation you only make once, or a donation that you make annually, but have not documented in an agreement. You may deduct your ordinary donation if it meets the following conditions:

Reviewer: ```suggestion
An ordinary donation is a donation you only make once, or a donation that you make annually, but have not documented in an agreement. You may deduct your ordinary donation if it meets the following conditions:
```

* Your donation will go to an ANBI (public benefit organization)
* Your donation is voluntary
* You get nothing in exchange for your donation
* You can prove that you have made the gift. This can be seen, for example, from the withdrawals from your account

> [Read more about tax deductability](https://www.belastingdienst.nl/wps/wcm/connect/nl/aftrek-en-kortingen/content/gift-aftrekken) for donations from the Dutch tax office (in Dutch).

Reviewer: ```suggestion
[Read more about tax deductibility](https://www.belastingdienst.nl/wps/wcm/connect/nl/aftrek-en-kortingen/content/gift-aftrekken) for donations from the Dutch tax office (in Dutch).
```

> [Read more about tax deductability](https://www.belastingdienst.nl/wps/wcm/connect/nl/aftrek-en-kortingen/content/gift-aftrekken) for donations from the Dutch tax office (in Dutch).

Reviewer: ```suggestion
[Read more about tax deductibility](https://www.belastingdienst.nl/wps/wcm/connect/nl/aftrek-en-kortingen/content/gift-aftrekken) for donations from the Dutch tax office (in Dutch).
```
